README.md

```diff
@@ -108,7 +115,148 @@ Log in with psql shell to cnx-db
 
     make sql
 
+### S3 access for debugging
+
+You need opensource software [rclone][rclone] for this guide. rclone enables accessing various cloud storage providers in a way similar to rsync. It also let's you mount s3 with FUSE.
```

Comment:
Do we need to include a link to install FUSE?

Reply:
Additionally on OSX you can install both with:
brew cask install FUSE
brew install rsync

Reply:
Added more information in README, I mentioned this link later in the document. For FUSE I personally dislike homebrew cask in this case (but only because the normal installer is doing a perfect job in updating and installing this system component).